README.md

```diff
@@ -6,29 +6,31 @@
 
 <!--
     Copyright 2019 Joyent, Inc.
+    Copyright 2024 MNX Cloud, Inc.
 -->
 
 # triton-grafana
 
 The Triton core grafana service. Triton uses
-[Prometheus](https://github.com/joyent/triton-prometheus) and
+[Prometheus](https://github.com/TritonDataCenter/triton-prometheus) and
 Grafana to track its own metrics and monitor itself, as well as Manta. All
-metrics are gathered via [CMON](https://github.com/joyent/triton-cmon).
+metrics are gathered via [CMON](https://github.com/TritonDataCenter/triton-cmon).
 
-(This repository is part of the Joyent Triton project. For general documentation
-see the main [Triton project](https://github.com/joyent/triton) page.)
+(This repository is part of the Triton Data Center project. For general
+documentation see the main [Triton project](https://github.com/TritonDataCenter/triton)
+page.)
 
 ## Status
 
-Joyent is actively developing Prometheus and Grafana services for use in Triton
-and Manta. [RFD 150](https://github.com/joyent/rfd/tree/master/rfd/0150)
+We are actively developing Prometheus and Grafana services for use in Triton
+and Manta. [RFD 150](https://github.com/TritonDataCenter/rfd/tree/master/rfd/0150)
 describes the current plan and status.
 
 ## Setup
 
 Grafana requires either a Triton or Manta Prometheus instance to exist, and will
 also function with both simultaneously. Ensure that
-[Prometheus](https://github.com/joyent/triton-prometheus) is set up in your
+[Prometheus](https://github.com/TritonDataCenter/triton-prometheus) is set up in your
```

Comment:
Maybe not now, but given our current `triton-prometheus` situation, perhaps this needs fixing later?